src/utils.rs

```diff
@@ -287,6 +304,10 @@ fn internal_exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
                 stderr: stderr_string,
             })
         }
+        Ok(None) => {
+            log::warn!("Executing command {:?} timedout", cmd);
```

Comment:
I think this needs to separated with a space:
```suggestion
            log::warn!("Executing command {:?} timed out", cmd);
```